metadata-ingestion/src/datahub/ingestion/run/pipeline_config.py

```diff
@@ -57,6 +57,11 @@ class FlagsConfig(ConfigModel):
         ),
     )
 
+    auto_lowercase_urns: bool = Field(
```

Comment:
I would expect this to be part of the `SourceConfig` class

Reply:
Yup agreed - let's do this on a per-source basis for now